packages/enzyme/src/_helpers/getTagNameFromHTMLConstructor.js

```diff
@@ -0,0 +1,97 @@
+
+/* eslint-env browser */
+/**
+ * map from constructor to tagName
+ */
+export default function getTagNameFromHTMLConstructor() {
+  return {
+    HTMLAnchorElement: a,
```

Comment:
these have to be strings - ie:
```suggestion
    HTMLAnchorElement: 'a',
```
etc